docs/UserGuide.md

```diff
@@ -1,134 +1,125 @@
-Format: `find KEYWORD [MORE_KEYWORDS]`
-
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
-
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
-
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
-
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
-
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+
+#### Adding a task: `add`
+Adds a task to the task-list.<br>
+Different Types of tasks (Todo/Event/Deadline) have different command format.<br>
+
+###### Adding a Todo:
+Todos will be rearranged in the Todo-List based on their priority.<br>
+> Format: `add TASK_NAME p/PRIORITY`<br>
+> Example: `add Assignment 3 p/1`
+
+###### Adding an Event:
+> Format: `add TASK_NAME d/DATE s/START_TIME e/END_TIME`<br>
+> Example: `add Time's birthday party d/25-12-2016 s/1400 e/1600`
+
+###### Adding a Deadline:
+> Format: `add TASK_NAME d/DATE e/END_TIME`<br>
+> Example: `add CS2103 v0.2 d/25-12-2016 e/1600`
+
+#### Editing a task: `edit`
+Edits information of the task in the task-list.<br>
+Different Types of tasks (Todo/Event/Deadline) have different command format.<br>
+
+###### Editing a Todo:
+Todos will be rearranged in the Todo-List based on their priority.<br>
+> Format: `edit TASK_TYPE INDEX_NUMBER n/TASK_NAME p/PRIORITY`<br>
+> Example: `edit todo 1 n/Assignment 1 p/2`
+
+###### Editing an Event:
+> Format: `edit TASK_TYPE INDEX_NUMBER n/TASK_NAME d/DATE s/START_TIME e/END_TIME`<br>
+> Example: `edit event 1 n/Time's birthday party d/25-12-2016 s/1200 e/1600`
```

Comment:
you see this becomes a pain point to the user. if he wants to change just the time, he has to enter the entire event details again in your application's current form,